probe/endpoint/conntrack_test.go

```diff
@@ -0,0 +1,138 @@
+package endpoint_test
+
+import (
+	"bufio"
+	"encoding/xml"
+	"io"
+	"testing"
+	"time"
+
+	. "github.com/weaveworks/scope/probe/endpoint"
```

Comment:
Ah, that page gives good guidance: only use it when there is a circular dependency issue in foo_test packages. Do we have a circular dependency here?